spec/live/cbp_support_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'core/spec_helper'
+require 'core/resources/cbp_spec_helper'
+
+describe 'Endpoints that support CBP' do
+  describe ZendeskAPI::Group do
+    describe '/groups' do
```

Comment:
I think RSpec is about describing `Object#methods`. This makes the test suite into documentation. IE: `describe "#fetch"`

https://www.betterspecs.org/#describe